tooling/nargo_cli/src/cli/test_cmd.rs

```diff
@@ -93,9 +97,17 @@ pub(crate) struct TestCommand {
     #[arg(long)]
     fuzzing_failure_dir: Option<String>,
 
-    /// Maximum time in seconds to spend fuzzing (default: 1 second)
+    /// Maximum time in seconds to spend fuzzing (default: 1 seconds)
     #[arg(long, default_value_t = 1)]
     fuzz_timeout: u64,
```

Comment:
Should this be changed to 10 seconds?

Reply:
No, I left it as 1, because it blew up the timing on aztec.nr.